bodhi/server/models.py

```diff
@@ -61,6 +60,135 @@
 except ImportError:
     log.warning("Could not import 'rpm'")
 
+import re
+
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
```

Comment:
Do I have to write this in place of https://github.com/ankit01ojha/bodhi/blob/0ce799e298fa85d3f734debd4a5b439a50275a3a/bodhi/server/models.py#L45 or before it?